components/FlexibleHeader/examples/supplemental/FlexibleHeaderWrappedSupplemental.m

```diff
@@ -21,8 +21,6 @@
 
 #import "FlexibleHeaderWrappedSupplemental.h"
 
-#import "MaterialFlexibleHeader.h"
```

Comment:
interesting. how did you detect that the component wasn't being used?

Reply:
I have no idea. It's been so long ago.